src/test/java/uk/ac/ebi/ampt2d/accession/file/UuidFileAccessioningServiceTest.java

```diff
@@ -17,35 +17,23 @@
  */
 package uk.ac.ebi.ampt2d.accession.file;
 
-import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.test.context.ContextConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.Arrays;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
 
 @RunWith(SpringRunner.class)
-@ContextConfiguration(classes = FileTestConfiguration.class)
-@DataJpaTest
+@SpringBootTest
```

Comment:
It would be nice to test as many classes as possible without making them dependent on the Spring context. For instance, we could construct an `UuidFileAccessioningService` without injecting anything, just by providing the constructor arguments on the fly.